rpcserver.go

```diff
@@ -1580,6 +1575,16 @@ func (r *rpcServer) OpenChannel(in *lnrpc.OpenChannelRequest,
 			"wallet is fully synced")
 	}
 
+	return nil
+}
+
+func (r *rpcServer) parseOpenChannelReq(
+	in *lnrpc.OpenChannelRequest) (*openChanReq, error) {
+
+	rpcsLog.Debugf("[openchannel] request to NodeKey(%v) "+
```

Comment:
Nit: now this won't be logged if `canOpenChannel` returns false, so we'd want to move this elsewhere.

Reply:
indeed, i think it's fine tho. we have an interceptor that logs errors and trace isn't really meant for production anyway